src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.scss

```diff
@@ -5,7 +5,7 @@
     flex-direction: column;
     height: 100%;
     position: relative;
-    width: 100%;
+    width: 340px;
```

Comment:
Do we have a fixed size for everything sidebar related?

Reply:
Yeah, 60 for nav, 340 for the content, so total 400